misc/log4j/src/main/java/org/openscience/cdk/tools/LoggingTool.java

```diff
@@ -170,13 +193,20 @@ public LoggingTool(Class<?> classInst) {
      */
     public static void configureLog4j() {
         LoggingTool localLogger = new LoggingTool(LoggingTool.class);
-        try { // NOPMD
-            org.apache.log4j.PropertyConfigurator.configure(LoggingTool.class
-                    .getResource("/org/openscience/cdk/config/data/log4j.properties"));
+        try {
+            try (InputStream resourceAsStream = LoggingTool.class.getResourceAsStream("cdk-log4j2.xml")) {
+                if (resourceAsStream != null) {
+                    XmlConfiguration config = new XmlConfiguration(
+                            LoggerContext.getContext(true),
+                            new ConfigurationSource(resourceAsStream));
+                    Configurator.reconfigure(config);
+                }
+            }
         } catch (NullPointerException e) { // NOPMD
             localLogger.error("Properties file not found: ", e.getMessage());
             localLogger.debug(e);
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Is this intended?

Reply:
(can be fixed later if not)

Reply:
Good spot - was fighting log4j2 config loading. The doc is very bad